### DISCUSSION TITLE
É possível baixar xml's referentes às notas ficais de entrada

### DISCUSSION THREAD

**Discussion by LCASAGRA1** [TadaSoftware/PyNFe#217] [General]
Sou novo no estudo do PyNFE. Pelo que verifiquei é um projeto bem completo.

É possível baixar os xml's referentes às NFE de entrada para um determinado xml?

Se sim poderia disponibilizar um exemplo de código?

Agradeço antecipadamente

**Comment by leogregianin** [comment]
Confere os exemplos:
https://github.com/TadaSoftware/PyNFe/wiki/Consulta-Distribui%C3%A7%C3%A3o
https://github.com/TadaSoftware/PyNFe/wiki/Fluxo-da-Consulta-de-Distribuicao-por-NSU

**Comment by leogregianin** [comment]
@LCASAGRA1, para melhor organização das discussões do projeto, pode marcar essa discussão por respondida?
